tasks/main.yml

```diff
@@ -1,50 +1,14 @@
 # SPDX-FileCopyrightText: 2022 Alexander Dahl <alex@netz39.de>
 # SPDX-License-Identifier: MIT
 ---
-# tasks file for telegraf-docker-in-docker
+# main tasks file for telegraf-docker-in-docker
 
-- name: Ensure system user is present on the host and in group docker
-  ansible.builtin.user:
-    name: "{{ tdid_user }}"
-    system: true
-    append: true
-    groups: docker
+- name: Run all tasks for role telegraf-docker-in-docker
```

Comment:
nit: This name is very non-descriptive - running the tasks is expected from the playbook anyways. I would focus on the condition here, like: "Check if role can be applied to the target host"